tests/h/views/api/users_test.py

```diff
@@ -12,7 +12,26 @@
 from h.services.user_signup import UserSignupService
 from h.services.user_update import UserUpdateService
 from h.services.user_unique import UserUniqueService, DuplicateUserError
-from h.views.api.users import create, update
+from h.views.api.users import read, create, update
+
+
+class TestRead(object):
+    def test_it_presents_user(
+        self, pyramid_request, fakeUserContext, TrustedUserJSONPresenter
+    ):
+        read(fakeUserContext, pyramid_request)
+
+        TrustedUserJSONPresenter.assert_called_once_with(fakeUserContext.user)
```

Comment:
Unfortunately you also need to test that `asdict()` was called correctly. I think this is probably a design problem with the presenters layer. But for now:

```suggestion
        TrustedUserJSONPresenter.assert_called_once_with(fakeUserContext.user)
        TrustedUserJSONPresenter.return_value.asdict.assert_called_once_with()
```